pushPackage.py

```diff
@@ -0,0 +1,29 @@
+import os
+import subprocess
+import shutil
+
+packagesPath = "/path/to/existed/package/directory"
```

Comment:
I'd rather make this script an "official" command on the BaGet command line. Something like `baget import /path/to/local/feed`.

Would you mind removing the script from this change so that we can fix the upload limit right away? 😄